Deprecated: fontMetrics is deprecated in PyQt5

```
AnyQt               0.2.0
PyQt5               5.15.10
PyQt5-Qt5           5.15.2
PyQt5-sip           12.13.0
```

```python
src/orangecontrib/ewoksxrpd/integrate1d.py:27: in _init_main_area
    self._plot = Plot1D()
/usr/local/lib/python3.7/site-packages/silx/gui/plot/PlotWindow.py:896: in __init__
    fit=True,
/usr/local/lib/python3.7/site-packages/silx/gui/plot/PlotWindow.py:273: in __init__
    self._positionWidget = tools.PositionInfo(plot=self, converters=converters)
/usr/local/lib/python3.7/site-packages/silx/gui/plot/tools/PositionInfo.py:131: in __init__
    contentWidget = _PositionInfoLabel(self)
/usr/local/lib/python3.7/site-packages/silx/gui/plot/tools/PositionInfo.py:53: in __init__
    super().__init__(*args, **kwargs)
/usr/local/lib/python3.7/site-packages/silx/gui/widgets/ElidedLabel.py:52: in __init__
    self.__updateMinimumSize()
/usr/local/lib/python3.7/site-packages/silx/gui/widgets/ElidedLabel.py:66: in __updateMinimumSize
    width = metrics.width("...")
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <PyQt5.QtGui.QFontMetrics object at 0x7f30a1ea99d0>, args = ('...',)
kwargs = {}
    def QFontMetrics_width(self, *args, **kwargs):
        warn("QFontMetrics(F).width is obsolete. "
             "Replace with QFontMetrics(F).horizontalAdvance",
>            DeprecationWarning, stacklevel=2)
E       DeprecationWarning: QFontMetrics(F).width is obsolete. Replace with QFontMetrics(F).horizontalAdvance
/usr/local/lib/python3.7/site-packages/AnyQt/QtGui.py:391: DeprecationWarning
```

For pyqt6 we already uses horizontalAdvance: https://github.com/silx-kit/silx/commit/1da99beb5d0585735ebe0b4977161e40978f3bd8

--- a/src/silx/gui/widgets/ElidedLabel.py
+++ b/src/silx/gui/widgets/ElidedLabel.py
@@ -1,6 +1,6 @@
 # /*##########################################################################
 #
-# Copyright (c) 2004-2021 European Synchrotron Radiation Facility
+# Copyright (c) 2004-2024 European Synchrotron Radiation Facility
 #
 # Permission is hereby granted, free of charge, to any person obtaining a copy
 # of this software and associated documentation files (the "Software"), to deal
@@ -27,6 +27,8 @@
 __license__ = "MIT"
 __date__ = "07/12/2018"
 
+
+from packaging.version import Version
 from ...utils.deprecation import deprecated
 from silx.gui import qt
 
@@ -62,7 +64,7 @@ class ElidedLabel(qt.QLabel):
 
     def __updateMinimumSize(self):
         metrics = self.fontMetrics()
-        if qt.BINDING == "PyQt5":
+        if Version(qt.qVersion()) < Version("5.11"):  # PyQt5
             width = metrics.width("...")
         else:  # Qt6
             width = metrics.horizontalAdvance("...")

